ctx_test.go

```diff
@@ -1412,6 +1416,18 @@ func Benchmark_Ctx_Fresh_WithNoCache(b *testing.B) {
 	}
 }
 
+// go test -v -run=^$ -bench=Benchmark_Ctx_Fresh_LastModified -benchmem -count=4
+func Benchmark_Ctx_Fresh_LastModified(b *testing.B) {
+	app := New()
+	c := app.AcquireCtx(&fasthttp.RequestCtx{})
+
+	for n := 0; n < b.N; n++ {
+		c.Response().Header.Set(HeaderLastModified, "Wed, 21 Oct 2015 07:28:00 GMT")
```

Comment:
These Header.Set should go outside the loop. Only `c.Fresh()` should be inside.

Reply:
Corrected according to your suggestion. Also, the benchmark scores in the PR comments have been updated.